app/controllers/orders_controller.rb

```diff
@@ -24,6 +26,30 @@ def show
 
   private
 
+  def send_photographer_email(order)
+    photos = order.order_photos.map { |order_photo| Photo.find(order_photo.photo_id) }
+    store_ids = photos.map { |photo| photo.store_id }.uniq
+    stores = store_ids.map { |id| Store.find(id) }
+    stores.each do |store|
+      NotificationsMailer.contact(
+        name: "Imagine Photography",
+        email: "#{store.email}",
+        message: "You have received an order for one of your photos! You can
```

Comment:
instead of stating "one of your photos", let's actually interpolate the photos count for each store into the message.